tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
tsconfig nie zaczyna być przypadkiem deprecated?
https://code.visualstudio.com/api/advanced-topics/tslint-eslint-migration

Reply:
A nie wiedziałem o tym 😄 Może vite.js dodaje go z automatu?

Reply:
moze jakieś standardowe ustawienia - jak działa to zostaw, ale z tego co wiem to już eslint zaczyna przejmować TSowe rzeczy i działa szybciej i lepiej :)

Reply:
Good to know :)